app/views/issue_mailer/status_changed.html.erb

```diff
@@ -0,0 +1,2 @@
+<h3>Status for Issue, '<%= @issue.title %>' Updated</h3>
+<p>Status for the issue '<%= link_to @issue.title, project_issue_url(@issue.project_id, @issue.id, subdomain: @company.subdomain) %>' updated to '<%= Issue.human_enum_name(:status, @issue.status) %>'</p>
```

Comment:
why do we need to pass subdomain?

Reply:
so that the url will have subdomain like acompany.lvh.me:3000